### ISSUE TITLE
Loading http://localhostsettings.cgi/

### ISSUE THREAD

**Issue by Thanosgrc** [dobey/diogenes-ng#3]
When I try to load the databases of Greek/Latin and also the Inscription's clicking on «current settings page» and i give the exact location (e.g. /home/test/Desktop/TLG next to "The location of the TLG database: etc") and I try to save those changes it redirects me to "http://localhostsettings.cgi/" and game over.

> We can’t connect to the server at localhostsettings.cgi.
> If that address is correct, here are three other things you can try:
> 
>     Try again later.
>     Check your network connection.
>     If you are connected but behind a firewall, check that Firefox has permission to access the Web. 

You may find those databases on TPB searching "TLG". Those are the sources that Diogenes reads from and do all the work (PHI5,7 And TLG-E). 
PS. I'm using sudo with this consumable VM because it was pointing out more stuff. I need this for work and research on my field, I should thank YOU for your time, as Benjamin Franklin said "Do you love life? Then do not squander time, for that’s the stuff life is made of." If you manage to fix some more things I will test it in depth and let you know because some people need this like me. Thanks again. Also I never prompted the link, it would be a good idea for such a notification.

Edit: I uploaded the DB [here](http://www.mediafire.com/file/jbcabyfc2vol2cw/Tlg.zip/file).

**Comment by isness** [comment]
Trying to install Diogenes on Ubuntu 17.10 I get the same problem. It would be wonderful if you had time to fix this issue.

**Comment by jamiedow** [comment]
Hi, I've got this issue too. It's a showstopper.
It's just possible it may be possible to fix the issues manually by posting something like this into a file called either diogenes.prefs or diogenes.config (or possibly one of those with diogenes-ng instead of diogenes), in the folder ~/.diogenes-ng ... but I haven't tried that ... I'll try giving it a go now.

context "sentence"
cgi_default_encoding "UTF-8"
perseus_show "split"
cgi_font "Gentium Plus"
browse_lines "29"
input_encoding "Unicode"
tlg_dir "/home/jamie/Documents/TLG/Tlg/TLG-E (Greek Texts)/"
phi_dir "/home/jamie/Documents/TLG/Tlg/PHI-5 (Latin texts)/"
ddp_dir ""

**Comment by jamiedow** [comment]
Yeah, tried it. Hard to tell whether it worked because when searching, it doesn't look as though the search button is really working either. It seems as though all of the button-involving operations from either the search page or the settings page involve some kind of localhostsettings error.

Help!

**Comment by On3simos** [comment]
Hello,
I managed to make the original version of Diogenes on Xubuntu 16.04 and 18.04 with only a few steps to amend the code. See my answer to the post : https://askubuntu.com/questions/1065671/how-to-run-diogenes-app-on-modern-ubuntu
It seems much better than this, which I did not manage to make work.
